docs/api/carts.rst

```diff
@@ -210,6 +207,7 @@ Exemplary Response
 .. note::
 
     A currency code will be added automatically based on the channel settings. Read more about channels :doc:`here </book/configuration/channels>`.
+    Same rule applies locale codes. Locale can be changed during complete request when finalizing a cart.
```

Comment:
`(...) to locale codes.`,
The last sentence sounds a whole lot like a pleonasm. Better phrasing would be much appreciated.